test/models/bot/alegre_2_test.rb

```diff
@@ -526,6 +526,36 @@ def teardown
     assert_equal({}, Bot::Alegre.get_items_from_similar_text(random_number, ''))
   end
 
+  test "get_items_from_similar_texts should not search bad titles" do
+    text = "instagram-cekfakta-2023-25562004"
+    pm1 = create_project_media team: @team, quote: text
+    Bot::Alegre.stubs(:request).raises("Request method called when it should not be")
+    assert_nothing_raised do
+        Bot::Alegre.get_items_from_similar_text(@team, text)
+    end
+    Bot::Alegre.unstub.stubs(:request)
+  end
+
+  test "/text/similarity/search/ request should be called for good title" do
+    text = "This should pass to Alegre"
+    pm1 = create_project_media team: @team, quote: text
+    Bot::Alegre.stubs(:request).raises("Request method called when it should not be")
+    assert_raises do
+        Bot::Alegre.get_items_from_similar_text(@team, text)
+    end
+    Bot::Alegre.unstub.stubs(:request)
+  end
+
+  test "/text/similarity/ request should be called for good title" do
```

Comment:
This method should be in `alegre_4_test.rb` with the other tests that use `send_to_text_similarity_index`